demos/smartlab_online_demo/python/README.md

```diff
@@ -0,0 +1,111 @@
+
+## How It works
+
+The demo pipeline consists of several steps, namely 'Decode', 'Detector', 'Segmentor', 'Evaluator', 'Display'.
+Every step is implemeted by correcponding class. Decode is implemented by opencv API, Detector is imlemeted `detector.py`. Segmentor is implemented by `segmentor.py`. Evaluator is implemented by `segmentor.py`. Evaluator is implemented by `evaluator.py`. Display is implemneted by `display.py`
+
+* `Decode` reads frames from the input videos.
+* `Detector` detects smartlab objects (balance, weights, tweezers, box, battery, tray, ruler, rider scale, hand)
+* `Segmentor` segment video frames based on action of the frame.
+* `Evaluator` calculates scores of the current state.
+* `Display` display detected objects, regnized action, calculated scores on the current frame.
+
+
+> **NOTE**: By default, Open Model Zoo demos expect input with BGR channels order. If you trained your model to work with RGB order, you need to manually rearrange the default channels order in the demo application or reconvert your model using the Model Optimizer tool with the `--reverse_input_channels` argument specified. For more information about the argument, refer to **When to Reverse Input Channels** section of [Converting a Model Using General Conversion Parameters](https://docs.openvino.ai/latest/openvino_docs_MO_DG_prepare_model_convert_model_Converting_Model.html#general-conversion-parameters).
+
+## Preparing to Run
+For demo input image or video files, need to provide smartlab videos following our setup (any shareable URL???).
+The list of models supported by the demo is in `<omz_dir>/demos/smartlab_online_demo/python/models.lst` file.
+This file can be used as a parameter for [Model Downloader](../../../tools/model_tools/README.md) and Converter to download and, if necessary, convert models to OpenVINO Inference Engine format (\*.xml + \*.bin).
+
+An example of using the Model Downloader:
+
+```sh
+omz_downloader --list models.lst
+```
+
+An example of using the Model Converter:
+
+```sh
+omz_converter --list models.lst
+```
+
+### Supported Models
+smartlab-action-recognition-encoder-0001
+smartlab-action-recognition-decoder-0001
+smartlab-object-detection-0001
+smartlab-object-detection-0002
+smartlab-object-detection-0003
+smartlab-object-detection-0004
+
+> **NOTE**: Refer to the tables [Intel's Pre-Trained Models Device Support](../../../models/intel/device_support.md) and [Public Pre-Trained Models Device Support](../../../models/public/device_support.md) for the details on models inference support at different devices.
+
+## Running
+
+Running the application with the `-h` option yields the following usage message:
+
+```
+usage: video_processor_serial.py [-h] -tv TOPVIEW -fv FRONTVIEW -m_ta M_TOPALL -m_tm M_TOPMOVE -m_fa M_FRONTALL -m_fm M_FRONTMOVE {multiview,mstcn} ...
+
+positional arguments:
+  {multiview,mstcn}     sub-command help
+    multiview           multiview help
+    mstcn               mstcm help
```

Comment:
I think think you didn't intend this to be an arg

Reply:
Actually we have two type demos (multiview and mstcn) in the demo application.
Is it better to separate them?

Currently, I removed mstcn.